atlp-rwanda/wolves-bn-frontend
# #174772166 - Setup Travis CI, coveralls and code climate

#### What does this PR do?
Setup TravisCI, coveralls and code climate
#### Description of Task to be completed?
Setup TravisCI, coveralls and code climate
#### How should this be manually tested?
Clone the repository
run **npm run test:coveralls** to check the tests coverage. 
Check badges on Read me page
#### What are the relevant pivotal tracker stories?
#174772166
